src/Annotation/Admin.php

```diff
@@ -13,6 +13,14 @@
 
 namespace Sonata\AdminBundle\Annotation;
 
+// NEXT_MAJOR: remove this file
+
+@trigger_error(
+    'The '.__NAMESPACE__.'\Admin class is deprecated since sonata-project/admin-bundle 3.46.0 and will be removed in 4.0.'
```

Comment:
I'm supposed to do the replacement from 3.x to 3.46 (IMO the `.0`  is not needed since deprecations may only happen in new minor versions). That way you don't have to worry about how long this is going to take.

Reply:
Sure, I just copied the message from the `@deprecated` annotation. We should update that message too given this premise.